dace/libraries/blas/nodes/dot.py

```diff
@@ -435,13 +572,32 @@ class Dot(dace.sdfg.nodes.LibraryNode):
     dtype = dace.properties.TypeClassProperty(allow_none=True)
     vec_width = dace.properties.SymbolicProperty(allow_none=False, default=1)
 
-    def __init__(self, name, dtype=None, vec_width=1, *args, **kwargs):
+    partial_width = dace.properties.SymbolicProperty(allow_none=False,
```

Comment:
Can you add a help entry here to describe this

Reply:
A `desc` has been added